### ISSUE TITLE
Drop dependency on pygments

### ISSUE THREAD

**Issue by Raynos** [mbrevoort/docco-husky#1]
I'd be great if you could kill the external syntax highlighting dependency on python & pygments and replace it with a node library.

**Comment by mbrevoort** [comment]
Pygments is pretty awesome, but I agree. Perhaps [CodeMirror](http://codemirror.net/) or [highlight.js](http://jga.me/blog/2011/08/18/highlight.js).

**Comment by Raynos** [comment]
Codemirror looks overkill, highlight.js is more sensible

**Comment by rapidrapids** [comment]
+1

**Comment by PunkChameleon** [comment]
Agreed. This should be done in docco too -- this repo has a version without pygments, I believe - https://github.com/eliias/grunt-docco
